### DISCUSSION TITLE
update laravel10-11/livewire2-3 results to Uncaught SyntaxError: expected expression, got '<'

### DISCUSSION THREAD

**Discussion by AliKetencimen** [livewire/livewire#8750] [1. Help]
Hi , I updated my project from laravel10 to 11 and lievwire 2 to 3, after that any pages made with livewire throws an error Uncaught SyntaxError: expected expression, got '<'. more details [here](https://stackoverflow.com/questions/78853404/update-laravel10-11-livewire-2-3-error-with-scripts-uncaught-syntaxerror-expe)

I tried 1000 things now and can't figure out why livewire.js is not loaded at all, no error 404, but I can see the file's content going to url /livewire/livewire.js

**Comment by lrljoe** [comment]
You need to share proper errors, including the stack trace.

Share the full details of where it's throwing
Uncaught SyntaxError: expected expression, got '<'. 

Otherwise people aren't going to play the guessing game.

I'd suspect that you've broken your app.js or something, but who knows without the detail

Either that or you're trying to still bundle an older version of Alpine/Livewire, and inject at the same time.

It's impossible to know without the full error log

**Reply by AliKetencimen** [reply]
Hi, all I have in the console is Uncaught SyntaxError: expected expression, got '<' and it points to index.html 1 (this line : <!doctype html>) sorry but that's all I have, I think I wouldn't be stuck on this for almost 2 weeks and posted questions on multiple forums if I had help :/
Thanks anyways trying

Here is all my console when I load a page made with livewire
![image](https://github.com/user-attachments/assets/ab31ff71-be3e-440a-ad7a-b586e233b4e4)

Btw let's imagine it is that I am bundling old versions of livewire and alpine after updating them, do you have an idea how to detect that? So I can find a solution or a start of solution? Because in my network tab lievwire.js is not even loaded, going to url /livewire/livewire.js I see the content of the file so as said in the docs it should be able to auto load livewire.js right?
